packages/react-server-website/nginx.conf

```diff
@@ -1,14 +1,30 @@
 proxy_cache_path /tmp/nginx levels=1:2 keys_zone=react_server_zone:10m inactive=60m;
 
-# Redirect requests to port 80 to HTTPS.
+##############################################################################
+# Redirects
+##############################################################################
 server {
+    # Redirect requests to port 80 to HTTPS.
     listen 80 default_server;
     server_name react-server.io;
 
     return 301 https://$server_name$request_uri;
 }
 
 server {
+    # Redirect requests to https://*.react-server.io to
+    # https://react-server.io, except slack.react-server.io, which is handled
+    # separately below.
+    listen 443;
+    server_name *.react-server.io;
+
+    return 301 https://react-server.io$request_uri;
```

Comment:
Need to remember to add a server entry for a new domain here before we set up DNS to avoid accidental 301.